src/promptflow/promptflow/_sdk/_configuration.py

```diff
@@ -32,6 +32,8 @@ class Configuration(object):
     EXTENSION_EU_USER = "extension.eu_user"
     INSTALLATION_ID = "cli.installation_id"
     CONNECTION_PROVIDER = "connection.provider"
+    RECORDING_MODE = "cli.recording.mode"
```

Comment:
maybe we should not have the cli prefix. As recording works for both cli & sdk?

Reply:
Yes, let's make it.